index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Video App</title>
+  <link rel="stylesheet" href="./index.css" />
+</head>
+
+<body>
+
+  <div class="main">
+
+    <nav class="navbar">
+      <div class="brand">
+        <img src="./img/menu.png" alt="menu-icon" />
+        <span>Video App</span>
+      </div>
+
+      <div class="input-field">
+        <input class="searchBar" type="text" />
+        <img src="./img/search.png" alt="search-icon" />
+      </div>
+
+      <div class="menu-section">
+        <ul class="top-menu">
+
+          <li><a href=""><img src="./img/notification.png" alt="" /></a></li>
+          <li><a href=""><img src="./img/video-camera.png" alt="" /></a></li>
+          <li><a href=""><img class="avatar" src="https://source.unsplash.com/random" alt="" /></a>
+          </li>
+
+        </ul>
+
+      </div>
+    </nav>
+
+    <section class="main-section">
+
+      <div class="r-side">
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/oGYv3sCuxVQ" title="YouTube video player"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen></iframe>
+
+        <div class="title">
+          <h3>What does Earth sound like from space?</h3>
+        </div>
+
+          <ul class="icon-bar">
+
+            <li class="icon-cols">Like</li>
+            <li class="icon-cols">Dislike</li>
+            <li class="icon-cols">Save</li>
+            <li class="icon-cols">Share</li>
+            <li class="icon-cols"> More</li>
+
+          </ul>   
```

Comment:
I think it will be better to use the list tag instead of span tags there to conveniently structure your code and make styling easier 😃 👍

Reply:
The student have used ul and li tags for this part of the project. So I am wondering why you are requesting changes to use list tag instead of span tags :confused: Kindly ensure that you leave the comments under a line or block of code that fits the context the comment is about in order not to confuse the student. :pray:

Reply:
Please in response to the review, i had span tags on these lines but made commit changes after my review that is changed them to a list tag after the review has been made. that is the reason for the list  tag.

Reply:
- [ ] Kindly add a meaningful code to review and review that in this PR not shown before it was span; this project aims to learn how to do a review and learn HTML and CSS codes. In giving a code review, please be polite and clear about the inline comment you give feedback on, and explain why it required changing aspects.

Reply:
As previous reviewers requested please add more meaningful code lines and review them and be clear about why you are asking for change.
>Please in response to the review, i had span tags on these lines but made commit changes after my review that is changed them to a list tag after the review has been made. that is the reason for the list tag.

ꜛUnfortunately, you need to review the codes committed to this Pull Request. So, please explain why you are asking for a change, give a reason or pick another line of codes that has problems and review that. 👌